stdlib/public/core/UTF16.swift

```diff
@@ -469,6 +470,17 @@ func allASCIIBlock(at pointer: UnsafePointer<UInt16>) -> SIMD16<UInt8>? {
     ? unsafeBitCast(block, to: SIMD32<UInt8>.self).evenHalf : nil
 }
 #endif
+#else
+@_transparent var blockSize:Int { 1 }
+@_transparent
+func allASCIIBlock(at pointer: UnsafePointer<UInt16>) -> CollectionOfOne<UInt8>? {
```

Comment:
Once we can use `InlineArray` in the stdlib, this should be easier to convert into a SWAR-style thing that does 8 elements at a time.

…if we need to. I don't think anyone is really depending on UTF16 transcoding perf in an environment that can't use SIMD, but hey, it's a big world, who knows.